pkg/driver/helmdriver.go

```diff
@@ -28,8 +29,14 @@ var _ PackageDriver = (*helmDriver)(nil)
 
 func NewHelm(log logr.Logger) (*helmDriver, error) {
 	settings := cli.New()
-	cfg := &action.Configuration{}
-	err := cfg.Init(settings.RESTClientGetter(), settings.Namespace(),
+	//TODO: Allow configuring the client's credentials from a secret
+	//see: https://github.com/aws/eks-anywhere-packages/issues/20
+	client, err := registry.NewClient()
+	if err != nil {
+		return nil, fmt.Errorf("Error creating registry client while initializing helm driver: %w", err)
```

Comment:
nit: There's no need to prefix this with "Error".

Reply:
```suggestion
		return nil, fmt.Errorf("creating registry client while initializing helm driver: %w", err)
```

Reply:
Lower case too